smartHome/src/main/java/com/cnu2016/assignment02/Main.java

```diff
@@ -0,0 +1,35 @@
+package com.cnu2016.assignment02;
+import java.io.*;
+import java.util.*;
+public class Main {
+    public static ArrayList<Appliance> populate() {
+        ArrayList<Appliance> appliances = new ArrayList<Appliance>();
+        appliances.add(new Appliance(Appliance.Type.WATERHEATER));
+        appliances.add(new Appliance(Appliance.Type.AIRCONDITIONER));
+        appliances.add(new Appliance(Appliance.Type.COOKINGOVEN));
+        appliances.add(new Appliance(Appliance.Type.WATERHEATER));
+        return appliances;
+    }
+    public static void main(String args[]) throws IOException, BadScheduleException {
+        ArrayList<Appliance> appliances = populate();
+        ArrayList<Event> events = EventGenerator.readData("/projects/assignments/smartHome/input.txt", appliances);
```

Comment:
1. what if no file at given path ?
2. what if the text inside the file doesn't have correct format